runtime/near-vm-logic/tests/test_context.rs

```diff
@@ -0,0 +1,113 @@
+mod fixtures;
+use near_vm_logic::mocks::mock_external::MockedExternal;
+use near_vm_logic::mocks::mock_memory::MockedMemory;
+use near_vm_logic::{Config, VMContext, VMLogic};
+
+pub fn create_context() -> VMContext {
+    VMContext {
+        current_account_id: "alice".to_owned(),
```

Comment:
please switch `to_owned` to `to_string` for strings

Reply:
I don't have any strong opinion on this really, but there is some arguments for using `to_owned`:
https://users.rust-lang.org/t/to-string-vs-to-owned-for-string-literals/1441/6